tasks/task03/word_stats.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+texts = [
+   "Hello, World!",
+   "The world is mine",
+   "Hello, how are you?"
+]
+
+word_stats = {}
+line = 0
+
+def word_statistics(word_stats):
+    if(word not in word_stats):
+        word_stats[word] = {'word_count': 1,'line_number': line}
+    else:
+        word_stats[word]['word_count'] = word_stats[word]['word_count']+1
+
+def print_stats(word_stats):
+    print("word\tcount\tfirst line")
+    for i in word_stats:
+        print(f"{i}\t{word_stats[i]['word_count']}\t{word_stats[i]['line_number']}")
+
+for text in texts:
+    for word in re.sub('([^A-z^\s])+', '', text).lower().split():
+        word_statistics(word_stats)
```

Comment:
I meant another structure

`def find_all_words(text):`

`def count_all_words(text)` <- Contains a call  `find_all_words`

`def print_stats(text):` <- Contains a call `count_all_words`

main function call print_stats without looping.

`re.sub('([^A-z^\s])+', '', text).lower()` should be in variable